gnss_lib_py/parsers/precise_ephemerides.py

```diff
@@ -663,19 +522,15 @@ def multi_gnss_from_precise_eph(navdata, sp3_path, clk_path, \
         precise ephemerides from .sp3 and .clk files
```

Comment:
Rename the navdata to measurements (or something else that is more indicative of what these values are supposed to be). Also, remove references to Android derived dataset